sync2jira/main.py

```diff
@@ -293,6 +301,113 @@ def initialize_pr(config, testing=False):
     log.info("Done with github PR initialization.")
 
 
+def initialize_recent(config):
+    """
+    Initializes based on the recent history of datagrepper
+
+    :param Dict config: Config dict
+    :return: Nothing
+    """
+    # Query datagrepper
+    ret = query(categories=['github', 'pagure'], delta=int(600), rows_per_page=100)
```

Comment:
These categories are not working. @ralphbean can you confirm that this is the correct way to format the query. 

This is ready for review except for test cases & docs

Reply:
Also python3-rhmsg can't be found in DNF. This will have to be fixed for `Dockerfile.deploy`

Reply:
Update: Docs and test cases have been updated. All that is left is the dockerfile update.

Reply:
I think this is the right category, but you may need to specify them as separate `category=` arguments.

I.e. https://apps.fedoraproject.org/datagrepper/raw?category=github&category=pagure&rows_per_page=1&delta=127800